shuup/reporting/utils.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import datetime
+import decimal
+
+import babel
+# import dateparser
+import six
+from babel.numbers import format_currency
+from django.utils import timezone
+from django.utils.translation import get_language
+
+ERROR_MESSAGE = "Error fetching data"
+NO_DATA_MESSAGE = "No results"
+
+
+def get_empty_data(schema, data, message):
+    keys = [s.get("key") for s in schema]
+    return_data = {}
+    for key in keys:
+        return_data[key] = message
+    return {
+        "data": [return_data],
+        "start": data["start"],
+        "end": data["end"],
+    }
+
+
+def get_error_data(schema, sales_data):
+    if sales_data.get("data") == "error":
+        return get_empty_data(schema, sales_data, ERROR_MESSAGE)
+    if not sales_data.get("data")[0]:
+        return get_empty_data(schema, sales_data, NO_DATA_MESSAGE)
+
+
+def to_datetime(value):
```

Comment:
I don't think we need this either. Consider using `shuup.utils.dates:parse_date`